### PR TITLE
Test on mruby 1.3.0

### PR DESCRIPTION
**Pull request by hfm**

### CODE REVIEW

**Comment by hfm** [comment]
Hi @matsumotory -san ✋ I changed mrbgem.rake to allow to test on mruby 1.3.0. Plz review it.

File: mrbgem.rake [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-  spec.add_dependency('mruby-socket', :core => 'mruby-socket')
   spec.add_test_dependency('mruby-sprintf', :core => 'mruby-sprintf')
   spec.add_dependency('mruby-polarssl')
   spec.add_test_dependency('mruby-simplehttpserver')
+  spec.add_test_dependency('mruby-sleep')
+
+  # need mruby-socket or mruby-uv
+  if MRuby::Source::MRUBY_VERSION >= '1.4.0'
+    spec.add_dependency('mruby-io', core: 'mruby-io')
+    spec.add_dependency('mruby-socket', :core => 'mruby-socket')
```
Comment (matsumotory) [inline mrbgem.rake]: core: mruby-socket?

Comment (hfm) [reply mrbgem.rake]: I fixed by 16533f8

**Comment by matsumotory** [comment]
Thanks!

**Pull request merged**